src/pages/guides/javascript/how-to-create-an-nft-on-solana.md

```diff
@@ -0,0 +1,47 @@
+---
+title: How to Create an Nft On Solana
+metaTitle: How to Create an Nft On Solana
+description: Learn how to create an Nft on the Solana blockchain with Metaplex packages.
+# remember to update dates also in /components/guides/index.js
+created: '06-16-2024'
+updated: '06-18-2024'
+---
+
+Metaplex provides 3 different standards for creating Nft's on the Solana blockchain including **Core**, **Token Metadata**, and **Bubblegum**. Each standard and protocol provides unique advantages for projects and spread across the wide spectrum of minting and Nft requirements for projects.
```

Comment:
Based on offline discussion, we still need to change 'Nft' to 'NFT' throughout this PR.